xbmc/addons/PVRClient.cpp

```diff
@@ -730,6 +730,35 @@ PVR_ERROR CPVRClient::RenameChannel(const CPVRChannelPtr &channel)
   return retVal;
 }
 
+PVR_ERROR CPVRClient::IsRecordable(const EPG::CEpgInfoTag *tag, bool *isRecordable) {
+  if (!m_bReadyToUse)
+    return PVR_ERROR_SERVER_ERROR;
+
+  if (!m_addonCapabilities.bSupportsRecordings)
+    return PVR_ERROR_NOT_IMPLEMENTED;
+
+  PVR_ERROR retVal(PVR_ERROR_UNKNOWN);
+  try
+  {
+    EPG_TAG pvrTag;
+    time_t t;
+    tag->StartAsUTC().GetAsTime(t);
+    pvrTag.startTime = t;
+    tag->EndAsUTC().GetAsTime(t);
+    pvrTag.endTime = t;
+    pvrTag.iUniqueBroadcastId = tag->UniqueBroadcastID();
```

Comment:
Submitting a partially filled EPG_TAG "feels" not good (although working for your use case). I can imagine that addons could make their "isrecordable" decision based on other criteria contained in the EPG_TAG. Short, I suggest to fill the EPG_TAG completely here, like done here for recordings https://github.com/xbmc/xbmc/blob/master/xbmc/addons/PVRClient.cpp#L275